projects/openvswitch/json_parser_target.c

```diff
@@ -0,0 +1,70 @@
+#include "openvswitch/json.h"
```

Comment:
Super nit: Let's include AFL's simple JSON dict for this target https://cs.chromium.org/chromium/src/testing/libfuzzer/fuzzers/dicts/json.dict?q=json.dict&sq=package:chromium&dr

Don't know if this would be useful for the jsonrpc stuff though.

Reply:
How do I wget this file?

Reply:
It never changes, I would probably just download it, or you can wget it from here: https://raw.githubusercontent.com/mcarpenter/afl/master/dictionaries/json.dict

Reply:
Never mind, looks like you did this.